html/browsers/the-window-object/window-indexed-properties.html

```diff
@@ -20,12 +20,17 @@
 test(function() {
   window[0] = "foo";
   assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { value: "bar" }))
+  assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { get() { return "baz" } }))
+  assert_throws(new TypeError(), () => Object.defineProperty(window, 0, { set() { return "quz" } }))
```

Comment:
Oh, I see my issue. It is the object literal syntax `{ set x() { return "baz"; } }` that is a SyntaxError. In defineProperty, it's just a normal object with a key named "set", which has no syntactic requirements.